examples/prerecorded/Prerecorded.csproj

```diff
@@ -2,7 +2,7 @@
 
 	<PropertyGroup>
 		<OutputType>Exe</OutputType>
-		<TargetFramework>net8.0</TargetFramework>
+		<TargetFramework>net6.0</TargetFramework>

```

Comment:
This doesn't match the line below, 
`<PackageReference Include="Microsoft.Extensions.Http" Version="8.0.0" />`
Do the versions need to match?